code/backend/batch/utilities/helpers/AzureBlobStorageHelper.py

```diff
@@ -7,10 +7,34 @@
     ContentSettings,
     UserDelegationKey,
 )
+from azure.storage.queue import QueueClient, BinaryBase64EncodePolicy
 from .EnvHelper import EnvHelper
 from azure.identity import DefaultAzureCredential
 
 
+def connection_string(account_name: str = None, account_key: str = None):
+    return f"DefaultEndpointsProtocol=https;AccountName={account_name};AccountKey={account_key};EndpointSuffix=core.windows.net"
+
+
+def create_queue_client():
+    env_helper: EnvHelper = EnvHelper()
+    if env_helper.AZURE_AUTH_TYPE == "rbac":
+        return QueueClient(
+            account_url=f"https://{env_helper.AZURE_BLOB_ACCOUNT_NAME}.queue.core.windows.net/",
+            queue_name=env_helper.DOCUMENT_PROCESSING_QUEUE_NAME,
+            credential=DefaultAzureCredential(),
+            message_encode_policy=BinaryBase64EncodePolicy(),
+        )
+
+    else:
+        return QueueClient.from_connection_string(
+            conn_str=connection_string(
+                env_helper.AZURE_BLOB_ACCOUNT_NAME, env_helper.AZURE_BLOB_ACCOUNT_KEY
+            ),
+            queue_name=env_helper.DOCUMENT_PROCESSING_QUEUE_NAME,
+        )
```

Comment:
I think we're missing `message_encode_policy`

Reply:
That's true... I wonder why it worked when I tested it 🤔

Reply:
🤔